INSTALL

```diff
@@ -46,7 +46,7 @@ How to install PBS Pro using the configure script.
   For Debian systems you should run the following command as root:
 
     apt-get install expat libedit2 postgresql python postgresql-contrib sendmail-bin \
-      sudo tcl tk libical1a
+      sudo tcl tk libical2
```

Comment:
Thanks for the contribution. At one point, the old instructions were accurate. Do you happen to know if this change will fail with older versions of Debian? If so, which ones?

Reply:
I think it probably worked with jessie (oldstable) but I can't confirm (when I was searching debian packages this appeared like it was true). Anything before that I have no idea/it would take a lot more digging.

Reply:
(clarify): the old instructions would work with jessie but the new instruction/package would not

Reply:
Looks like Jessie will be officially supported until June 30, 2020. Source: https://wiki.debian.org/LTS/

I think it would be better if we provide two instructions for Debian, one for Jessie and one for Stretch.